File: app/src/main/java/com/quran/labs/androidquran/data/VerseRange.java
package com.quran.labs.androidquran.data;

public class VerseRange {
  public final int startSura;
  public final int startAyah;
  public final int endingSura;
  public final int endingAyah;
  public final int versesInRange;

  public VerseRange(int startSura, int startAyah, int endingSura, int endingAyah, int verses) {
    this.startSura = startSura;
    this.startAyah = startAyah;
    this.endingSura = endingSura;
    this.endingAyah = endingAyah;
    this.versesInRange = verses;
  }
}

File: common/data/src/main/java/com/quran/data/model/VerseRange.kt
package com.quran.data.model

data class VerseRange(
  @JvmField val startSura: Int,
  @JvmField val startAyah: Int,
  @JvmField val endingSura: Int,
  @JvmField val endingAyah: Int,
  @JvmField val versesInRange: Int
)
